### PR TITLE
Presentation: Fix package build:ci script.

### PR DESCRIPTION
**Pull request by MartynasStrazdas**

### CODE REVIEW

File: presentation/common/package.json [hunk 1]
```diff
@@ -24,8 +24,9 @@
     "build": "npm run -s ruleset-json-schema && npm run -s build:assets && npm run -s build:public && npm run -s build:cjs",
     "build:ci": "npm run -s build && npm run -s build:esm",
     "build:cjs": "tsc -p tsconfig.cjs.json",
-    "build:esm": "tsc -p tsconfig.esm.json",
+    "build:esm": "npm run -s build:assets:esm && tsc -p tsconfig.esm.json",
     "build:assets": "cpx \"./assets/**/*\" ./lib/cjs/assets && npm run -s pseudolocalize:assets",
+    "build:assets:esm": "cpx \"./assets/**/*\" ./lib/esm/assets && npm run -s pseudolocalize:assets",
```
Comment (saskliutas) [inline presentation/common/package.json:29]:
```suggestion
    "build:assets:esm": "cpx \"./assets/**/*\" ./lib/esm/assets",
```

File: presentation/common/package.json [hunk 2]
```diff
@@ -24,8 +24,9 @@
     "build": "npm run -s ruleset-json-schema && npm run -s build:assets && npm run -s build:public && npm run -s build:cjs",
     "build:ci": "npm run -s build && npm run -s build:esm",
     "build:cjs": "tsc -p tsconfig.cjs.json",
-    "build:esm": "tsc -p tsconfig.esm.json",
+    "build:esm": "npm run -s build:assets:esm && tsc -p tsconfig.esm.json",
     "build:assets": "cpx \"./assets/**/*\" ./lib/cjs/assets && npm run -s pseudolocalize:assets",
+    "build:assets:esm": "cpx \"./assets/**/*\" ./lib/esm/assets",
```
Comment (aruniverse) [inline presentation/common/package.json:29]: arent these assets only used for the backend, which is using cjs?

Comment (saskliutas) [reply presentation/common/package.json:29]: We made changes to remove localization from presentation-backend and for backwards-compatibility we added [function to presentation-common](https://github.com/iTwin/itwinjs-core/blob/b3fe72be0a8b0ff2e416f79b4afc8cef2a6d395f/presentation/common/src/presentation-common/LocalizationHelper.ts#L18) which loads json files from assests and returns english strings. So now these assets are used in presentation-common esm module. 

Should we use different approach?

Comment (aruniverse) [reply presentation/common/package.json:29]: Are you expecting presentation-backend to use the presentation-common esm module?

Comment (saskliutas) [reply presentation/common/package.json:29]: No, it looks like we are building presentation-backend only as cjs.

Comment (aruniverse) [reply presentation/common/package.json:29]: only presentation-frontend would presentation-common esm, and if youre hoping to use the assets on the frontend, you need to make sure the app copies those over. by default we only copy the public folder with cra and i dont think we're gonna change that

Comment (saskliutas) [reply presentation/common/package.json:29]: That function is only used on backend so I am think about moving it from presentation-common to presentation-backend. Would something like this work in presentation-backend package: `require("@itwin/presentation-common/assets/locales/en/ECPresentation.json")`?

Comment (aruniverse) [reply presentation/common/package.json:29]: yes thatd probably work, but i think to keep things consitent, id just try:

`require("@itwin/presentation-common/lib/assets/locales/en/ECPresentation.json")`

or 

`require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json")`

the last one is the actual path, but i think the first is preferrable, would need to make sure the copy script gets updated and to check the npmignore file

Comment (saskliutas) [reply presentation/common/package.json:29]: Opened PR with changes: https://github.com/iTwin/itwinjs-core/pull/4190

**Pull request merged**